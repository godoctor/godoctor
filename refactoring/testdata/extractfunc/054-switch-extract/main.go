//<<<<< extract,9,2,14,2,newFunc,pass
package main

import "fmt"

func main() {
	s1 := "abcd"
	s2 := "efgh"
	switch s1 {
	case s2:
		fmt.Println("the two strings are equal")
	default:
		fmt.Println("the two strings are NOT equal")
	}
}
