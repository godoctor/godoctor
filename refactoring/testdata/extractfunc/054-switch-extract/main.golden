//<<<<< extract,9,2,14,2,newFunc,pass
package main

import "fmt"

func main() {
	s1 := "abcd"
	s2 := "efgh"
	newFunc(s1, s2)
}
func newFunc(s1 string, s2 string) {
	switch s1 {
	case s2:
		fmt.Println("the two strings are equal")
	default:
		fmt.Println("the two strings are NOT equal")
	}
}
